hpedockerplugin/cmd/cmd_createshare.py

```diff
@@ -18,28 +18,37 @@ def __init__(self, file_mgr, share_args):
         self._backend = file_mgr.get_backend()
         self._share_args = share_args
         self._status = 'CREATING'
+        self._share_created_at_backend = False
+        self._share_created_in_etcd = False
 
     def unexecute(self):
         share_name = self._share_args['name']
-        share = self._share_etcd.get_share(share_name)
         LOG.info("cmd::unexecute: Removing share entry from ETCD: %s" %
                  share_name)
-        self._etcd.delete_share(share_name)
-        if share['status'] == "AVAILABLE":
-            LOG.info("cmd::unexecute: Deleting share from backend: %s" %
-                     share_name)
+
+        # Leaving the share entry in ETCD intact so that user can inspect
+        # the share and look for the reason of failure. Moreover, Docker
+        # daemon has the entry for this share as we returned success on the
+        # main thread. So it would be better that the user removes this failed
```

Comment:
I am good with this change.

Reply:
Must have File Persona (xxx) license in Prerequisites.